skada/metrics.py

```diff
@@ -365,13 +366,36 @@ def _fit_adapt(self, features, features_target):
         if domain_classifier is None:
             domain_classifier = LogisticRegression()
         self.domain_classifier_ = clone(domain_classifier)
-        y_domain = np.concatenate((len(features) * [0], len(features_target) * [1]))
+        y_domain = np.concatenate((len(features) * [1], len(features_target) * [0]))
         self.domain_classifier_.fit(
             np.concatenate((features, features_target)), y_domain
         )
         return self
 
     def _score(self, estimator, X, y, sample_domain=None, **kwargs):
+        if not hasattr(estimator, "predict_proba"):
+            raise AttributeError(
+                "The estimator passed should have a 'predict_proba' method. "
+                f"The estimator {estimator!r} does not."
+            )
+
+        if not isinstance(estimator, BaseEstimator):
```

Comment:
Is this test very trustful? A non-deep model can still be a `BaseEstimator` no?

Reply:
Yes here there's 2 possibilities:
Either the estimator inherits from BaseEstimator or inherits from Nn.module.
But I wonder how it'll behave if we have a nn.module inside a sklearn.pipeline 🤔